styles/globals.css

```diff
@@ -1,139 +1,143 @@
 /* Colors */
 :root {
-    --primary: #ff5d22;
-    --text-primary: #212121;
-    --text-secondary: #757575;
+  --primary: #ff5d22;
+  --text-primary: #212121;
+  --text-secondary: #757575;
+  --header-background: #f2f2f7;
```

Comment:
I'm not sure if we should declare these colors manually.

Under the hood we use MUI, so these colors should come from it. Otherwise, if we will want to change it in future we will have to do it in at least two places (here, and in theme provider).

We could try to integrate their CSS theme variables: https://mui.com/material-ui/experimental-api/css-theme-variables/usage/

Reply:
You are right, it sucks. I tried to link you sent but couldn't make it work in the allotted time. I just deleted the variable and applied the same style using `sx`. I think we should tackle exposing theme variables as CSS variables in a separate task :)

Reply:
Sounds good, thank you!